infra/src/main/kotlin/fi/fta/geoviite/infra/geography/WktGeometries.kt

```diff
@@ -14,17 +16,14 @@ private const val LINESTRING_TYPE_3DM = "LINESTRING M"
 private const val POLYGON_TYPE_2D = "POLYGON"
 
 
-fun parse2DPoint(point: String): Point =
-    parse2DPointValues(dropParenthesis(dropTypeInfo(POINT_TYPE_2D, point)))
+fun parse2DPoint(point: String): Point = parse2DPointValues(dropWktType(point, POINT_TYPE_2D))
 
-fun parse2DLineString(lineString: String): List<Point> =
-    split2DPointValues(dropParenthesis(dropTypeInfo(LINESTRING_TYPE_2D, lineString)))
+fun parse2DLineString(lineString: String): List<Point> = split2DPointValues(dropWktType(lineString, LINESTRING_TYPE_2D))
 
 fun parse3DMLineString(lineString: String): List<Point3DM> =
-    split3DMPointValues(dropParenthesis(dropTypeInfo(LINESTRING_TYPE_3DM, lineString)))
+    split3DMPointValues(dropWktType(lineString, LINESTRING_TYPE_3DM))
```

Comment:
pikkuoptiomointina tehdään 2-osainen string leikkely kerralla. ei käytännön vaikutusta mutta olkoon näin päin